test/util.js

```diff
@@ -169,6 +169,18 @@ return {
 	/**
 	 * Etc
 	 */
+	assertRuntimeBind: function( assert, formatter, runtimeKey, generatorString, runtimeArgsFn ) {
+		assert.ok( formatter );
+		assert.equal( typeof formatter, "function" );
```

Comment:
The `formatter` can also be a "parser". Maybe rename to "actual", to reflect that this is what's being tested?